.ci/oci-e2e-deployment.sh

```diff
@@ -87,6 +87,9 @@ function executeE2ETests() {
 
 createQuayPullSecrets
 
+# create signing secret to test tekton-chains
+/bin/bash "$WORKSPACE"/hack/chains/create-signing-secret.sh
+
```

Comment:
Do we expect cosign to be installed already?

Reply:
Maybe a `command -v cosign ...` on line 23 for consistency?

Reply:
nope. Good catch. I just added it to the Dockerfile and a check this script.

Reply:
tekton-chains namespace does not exist yet. Should not be this rather in hack/bootstrap-cluster.sh since it should be set for all clusters? In case of putting it into bootrsap-cluster.sh I would do it only when cosign is installed (allow to continue without cosign)

Reply:
also possible to add into argo, probably as a PostSync job like `components/build/tekton-results/ingress-annotation-job.yaml`